src/specs/components/mega-menu/demo/data.js

```diff
@@ -128,7 +124,6 @@ module.exports = {
     {
       label: 'Key priorities',
       path: exampleLink,
-      link_aria_label: 'Link to Key priorities page',
```

Comment:
if we don't need anymore this, and it seems indeed that the data-ecl-parent-aria-label is not used anymore in the js, then we can also remove the logic that would render it: 
```
  {% if _item.link_aria_label is not empty %}
    {% set _menu_list_item_attributes = _menu_list_item_attributes ~ ' data-ecl-parent-aria-label="' ~ _item.link_aria_label ~ '"' %}
  {% endif %}
```
mega-menu-item.html.twig line 85 as well as:

```
          {% if child.link_aria_label %}
            {% set _subitem_attrs = _subitem_attrs ~ ' data-ecl-parent-aria-label="' ~ child.link_aria_label ~ '"' %}
          {% endif %}

line 217 and we don't need to pass it to the template:
`link_aria_label: child.link_aria_label|default(''),`
line 272
```
Also the docs are still mentioning the parameter